computation-local/src/main/java/com/powsybl/computation/local/LocalComputationConfig.java

```diff
@@ -63,17 +63,13 @@ private static Optional<Path> getTmpDir(ModuleConfig config, String name) {
     public static LocalComputationConfig load(PlatformConfig platformConfig, FileSystem fileSystem) {
         Objects.requireNonNull(platformConfig);
 
-        Path localDir = getDefaultLocalDir(fileSystem);
-        int availableCore = DEFAULT_AVAILABLE_CORE;
-        if (platformConfig.moduleExists(CONFIG_MODULE_NAME)) {
-            ModuleConfig config = platformConfig.getModuleConfig(CONFIG_MODULE_NAME);
-            localDir = getTmpDir(config, "tmp-dir")
-                           .orElseGet(() -> getTmpDir(config, "tmpDir")
-                                                .orElseGet(() -> getDefaultLocalDir(fileSystem)));
-            availableCore = config.getOptionalIntProperty("available-core")
-                                  .orElseGet(() -> config.getOptionalIntProperty("availableCore")
-                                                         .orElse(DEFAULT_AVAILABLE_CORE));
-        }
+        Optional<ModuleConfig> config = platformConfig.getOptionalModuleConfig(CONFIG_MODULE_NAME);
+        Path localDir = config.flatMap(c -> getTmpDir(c, "tmp-dir").or(() -> getTmpDir(c, "tmpDir")))
+                .orElse(getDefaultLocalDir(fileSystem));
+        int availableCore = config.map(c -> c.getOptionalIntProperty("available-core")
```

Comment:
and, worse, three `orElse`... as unfortunately there's no `OptionalInt::or(OptionalInt)` :disappointed: